examples/cactus-example-discounted-asset-trade/sign-utils.ts

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Wow, does HL fabric have documentation around this signing process? If yes, can you please attach some relevant fabric docs links about this, I would love to check them out.

Reply:
@jagpreetsinghsasan Sure, the process is described here very well - https://hyperledger.github.io/fabric-sdk-node/release-1.4/tutorial-sign-transaction-offline.html